app/Http/Controllers/ClassroomController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Classroom;
+use App\Models\Mentor;
+use Illuminate\Http\Request;
+
+class ClassroomController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    public function index()
+    {
+        $classrooms = Classroom::orderBy('id', 'ASC')->search()->get();
+        return view('classroom.index', compact('classrooms'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $mentor = Mentor::get();
+        return view('classroom.add', compact('mentor'));
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+//        $classroom_input = \request()->classroom_name;
+//        $clasroom_check = Classroom::insert([
+//            'classroom_name' => $request->classroom_name,
+//            'mentor_id' => $request->mentor_id,
+//            'roof' => $request->roof,
+//        ]);
+        Classroom::create($request->all());
+        $notification = [
+            'message' => __('text_message.classroom.create'),
+            'alert-type' => 'success',
+        ];
+        return redirect()->route('classrooms.index')->with($notification);
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param int $id
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $show = Classroom::where('id', $id)->first();
+        $mentor = Mentor::get();
+        $mentor_show = $show->mentor;
```

Comment:
don't need this variable. You can call relationship in blade

Reply:
fixed